### PR TITLE
Journal Card

### PR DESCRIPTION
**Pull request by yosefanajjar**
issue: #73 **Journal Card**

## What did I do?
- I made the layout for the journal card
- I made it ready for us to use props with it(add data to it)
- I made it responsive
- I downloaded the **Moment** module to help us format the time
- I wrote the shape of the object which comes or should come :smile: from the DB
- I wrote a storybook for the component so you can see it 

## How to test?
Just run `npm run storybook` and it should be with the components

**Note**: You can edit the object and work with moment to understand it

### CODE REVIEW

**Comment by Fatmasiam** [comment]
@yosefalnajjarofficial 
 # Write description related to what the component do not what you do

File: src/components/JournalCard/index.jsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import './style.css';
+
+const JournalCard = ({ time, date, grateful, challenges, developing }) => {
+  return (
+    <div className="journal-card">
+      <div className="journal-card__top">
+        <div className="journal-card__date">
+          <Icon
+            type="calendar"
+            style={{
```
Comment (Fatmasiam) [inline src/components/JournalCard/index.jsx]: remove inline-style try to use ```  className ```

File: src/components/JournalCard/style.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+@media (min-width: 600px) {
+  .journal-card  {
+    width: 60%;
+    margin: 0 auto;
+  }
+}
```
Comment (Fatmasiam) [inline src/components/JournalCard/style.css]: add new line

**Review by AlaaTaima — DISMISSED** [review]

**Review by fares98 — DISMISSED** [review]

File: src/components/JournalCard/style.css [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  width: 40%;
+}
+
+.journal-card__time {
+  width: 30%;
+}
+
+.journal-card__delete {
+  width: 30%;
+  display: flex;
```
Comment (Fatmasiam) [inline src/components/JournalCard/style.css:30]: what is the difference between line 21 and 29!

File: src/components/JournalCard/index.jsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        </div>
+      </div>
+      <div className="journal-card__body">
+        <p className="journal-card__grateful">
+          <span className="journal-card__body__title">grateful</span>
+          {grateful}
+        </p>
+        <p className="journal-card__challenges">
+          <span className="journal-card__body__title">challenge</span>
+          {challenges}
```
Comment (Fatmasiam) [inline src/components/JournalCard/index.jsx]:
```suggestion
          {challenge}
```

**Comment by Fatmasiam** [comment]
SOLVE THE CONFLICT !!!!

File: src/components/JournalCard/style.css (hunk 3 above)
Comment (yosefanajjar) [reply src/components/JournalCard/style.css:30]: the date contains text in it therefor it needs a bigger space on the card

**Review by fares98 — APPROVED** [review]

**Review by alaa-yasin — APPROVED** [review]

**Review by alaa-yasin — APPROVED** [review]

**Review by Fatmasiam — APPROVED** [review]

**Review by AlaaTaima — APPROVED** [review]

**Pull request merged**